CPRynner/CPRynner.py

```diff
@@ -3,130 +3,114 @@
     """
 
-    def __init__(self, cluster_address, tasks_per_node, work_dir, setup_script):
+    def __init__(self, cluster_address, tasks_per_node, work_dir, setup_script, run_command ):
         """Constructor"""
-        super().__init__(None, title="Login", size=(420, 480))
+        super(clusterSettingDialog, self).__init__(None, title="Login", size = (420,540))
 
         self.panel = wx.Panel(self)
 
         # cluster_address field
         cluster_address_sizer = wx.BoxSizer(wx.HORIZONTAL)
-        cluster_address_label = wx.StaticText(self.panel,
-                                              label="Cluster Address:",
-                                              size=(100, -1))
-        msg = (f"The URL of the cluster. If logged in, changing this will cause"
-               f" you to logout.")
-        cluster_address_label.SetToolTip(wx.ToolTip(msg))
-        cluster_address_sizer.Add(cluster_address_label, 0, wx.ALL | wx.CENTER,
-                                  5)
-        self.cluster_address = wx.TextCtrl(self.panel, value=cluster_address,
-                                           size=(300, -1), style=wx.TE_PROCESS_ENTER)
+        cluster_address_label = wx.StaticText(self.panel, label="Cluster Address:", size=(100, -1))
+        cluster_address_label.SetToolTip(wx.ToolTip(
+            "The URL of the cluster. If logged in, changing this will cause you to logout."
+        ))
+        cluster_address_sizer.Add(cluster_address_label, 0, wx.ALL|wx.CENTER, 5, )
+        self.cluster_address = wx.TextCtrl(self.panel, style = wx.TE_PROCESS_ENTER, value = cluster_address, size=(300, -1))
         cluster_address_sizer.Add(self.cluster_address, 0, wx.ALL, 5)
 
         # tasks_per_node field
         tasks_per_node_sizer = wx.BoxSizer(wx.HORIZONTAL)
-        tasks_per_node_label = wx.StaticText(self.panel,
-                                             label="Tasks Per Node:",
-                                             size=(300, -1))
-        msg = (f"The number of individual processes that can be run on a single"
-               f" node. Usually the number of cpu core per node.")
-        tasks_per_node_label.SetToolTip(wx.ToolTip(msg))
-        tasks_per_node_sizer.Add(tasks_per_node_label, 0, wx.ALL | wx.CENTER, 5)
-        self.tasks_per_node = wx.SpinCtrl(self.panel, value=str(tasks_per_node),
-                                          size=(100, -1))
+        tasks_per_node_label = wx.StaticText(self.panel, label="Tasks Per Node:", size=(300, -1))
+        tasks_per_node_label.SetToolTip(wx.ToolTip(
+            "The number of individual processes that can be run on a single node. Usually the number of cpu core per node."
+        ))
+        tasks_per_node_sizer.Add(tasks_per_node_label, 0, wx.ALL|wx.CENTER, 5)
+        self.tasks_per_node = wx.SpinCtrl(self.panel, value = str(tasks_per_node), size=(100, -1))
         tasks_per_node_sizer.Add(self.tasks_per_node, 0, wx.ALL, 5)
-
+        
```

Comment:
Empty lines should not have whitespace on; a linter or code formatter will help with this. (I won't comment on every instance, but please check all changes and make sure no blank lines are introduced.)